packages/grafana-data/src/valueFormats/categories.ts

```diff
@@ -39,6 +39,7 @@ export const getCategories = (): ValueFormatCategory[] => [
     name: 'Misc',
     formats: [
       { name: 'Number', id: 'none', fn: toFixedUnit('') },
+      { name: 'Decimal', id: 'decimal', fn: stringFormater },
```

Comment:
```suggestion
      { name: 'Number (Full precision)', id: 'decimal', fn: stringFormater },
```

The name Decimals seems to not really capture it I feel. 

Alt names
* Number (Raw)
* Number (Full precision)
* Raw (no formatting)